packages/trpc/server/routers/viewer.tsx

```diff
@@ -142,6 +146,112 @@ const publicViewerRouter = router({
     }),
   // REVIEW: This router is part of both the public and private viewer router?
   slots: slotsRouter,
+  event: publicProcedure
```

Comment:
New public query to fetch event data. We really have to double check this to ensure we're not leaking any data here that shouldn't be publicly queryable.